client/src/components/layout/BuildsList.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from "react"
+import BuildTile from "./BuildTile"
+
+
+const BuildsList = (props) => {
+
+    const [currentBuilds, setCurrentBuilds] = useState([])
+
+    const getBuilds = async () => {
+        try {
+            const response = await fetch("/api/v1/builds/")
+            const parsedResponse = await response.json()
+            setCurrentBuilds(parsedResponse.builds)
+        } catch (error) {
+            console.error(`Error Fetching Build List: ${error}`)
+        }
+    }
+
+    const buildTitles = currentBuilds.map((build) => {
+        return (
+            <BuildTile key={build.id} build={build} />
+        )
+    })
+
+    useEffect(() => {
+        getBuilds()
+    }, [])
+
+    return (
+        <div>
```

Comment:
this div also needs to go, but you do need a parent element. So use the empty `<>` fragment tag instead of <div>